package-lock.json

```diff
@@ -6165,7 +6166,6 @@
     },
     "node_modules/lodash": {
```

Comment:
Any real world project probably already has a dependency on lodash anyway, so this addition comes with no extra cost